### PR TITLE
Api

### PR DESCRIPTION
**Pull request by KashafLareb**
Add serializers and add permissions for comments

### CODE REVIEW

File: app/controllers/api/v1/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+class Api::V1::CommentsController < Api::V1::ApiController
+  before_action :authenticate
+  def index
```
Comment (Sultan0099) [inline app/controllers/api/v1/comments_controller.rb:4]: Add empty line before and after method

File: app/models/article.rb [hunk 2]
```diff
@@ -4,4 +4,19 @@ class Article < ApplicationRecord
   has_many :likes, dependent: :destroy
   has_rich_text :content
   validates :title, presence: true, uniqueness: true, length: { in: 5..30 }
+  def total_likes
+    likes.count
+  end
+  def liking_users
```
Comment (Sultan0099) [inline app/models/article.rb:12]: Add empty line before and after the method

File: app/controllers/api/v1/comments_controller.rb (hunk 1 above)
Comment (KashafLareb) [reply app/controllers/api/v1/comments_controller.rb:4]: okay

File: app/models/article.rb (hunk 2 above)
Comment (KashafLareb) [reply app/models/article.rb:12]: okay

**Pull request merged**